util/dvsim/Deploy.py

```diff
@@ -283,6 +286,18 @@ def get_timeout_mins(self):
         """Returns the timeout in minutes."""
         return 0
 
+    def get_runtimes(self, lines):
```

Comment:
Since this does not return anything, the `get_` prefix is a little misleading. Probably rename it to `extract_runtimes(log_text: List)` instead.